src/components/LayerSwitcher/AddCustomLayer.tsx

```diff
@@ -0,0 +1,283 @@
+import * as React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import { Button, CircularProgress, TextField } from '@material-ui/core';
+import { loadLayers } from '@dlurak/editor-layer-index';
+import { Autocomplete } from '@material-ui/lab';
+import { useAddLayerContext } from './helpers/AddLayerContext';
+
+type SuccessLayerDataInputProps = {
+  index: any[];
+  onSelect: (layer: any | undefined) => void;
+};
+
+const SuccessLayerInput: React.FC<SuccessLayerDataInputProps> = ({
+  index,
+  onSelect,
+}) => (
+  <Autocomplete
+    options={index.map((l) => l.name)}
```

Comment:
Do you think we should prefix the name with country code? Eg. `cz - IPR low vegetaion`? 
It could be useful for filtering. 

Also it could be cool to sort the list by country code and then by name.